frontend/src/Dashboard.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import LOCALIZE from "./text_resources";
+import ContentContainer from "./components/commons/ContentContainer";
+import { Helmet } from "react-helmet";
+
+const styles = {
+  title: {
+    color: "black"
```

Comment:
does this colour need to be different from our regular headings?

Reply:
I simply followed the design on that one:

![image](https://user-images.githubusercontent.com/23021242/57069423-99106100-6ca2-11e9-8a05-d8e914d8a706.png)

@caleybrock do you want me to let the default color or it is good like that?

Reply:
It looks like it's supposed to be black because it's supposed to be an h1 because it's the first heading on the page. Is that right @joeyhua ?

Reply:
Yeah, "Welcome to dashboard" should be an h1.

Reply:
Fixed in commit #https://github.com/code-for-canada/project-thundercat/pull/258/commits/006c9fa84bc83e0ec3084f5fcc81592b4d34b97a